Source/events/EventStoreBuilder.ts

```diff
@@ -1,131 +1,38 @@
 // Copyright (c) Dolittle. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-import { ChannelCredentials } from 'grpc';
-import { ArtifactsBuilder, ArtifactsBuilderCallback, IArtifacts } from '@dolittle/sdk.artifacts';
-import { EventHandlersBuilder, EventHandlersBuilderCallback, IEventHandlers, eventHandler } from '@dolittle/sdk.events.handling';
-import { EventFiltersBuilder, EventFiltersBuilderCallback, IFilters } from '@dolittle/sdk.events.filtering';
-import { EventHandlersClient } from '@dolittle/runtime.contracts/Runtime/Events.Processing/EventHandlers_grpc_pb';
-import { IContainer } from '@dolittle/sdk.common';
-import { IExecutionContextManager } from '@dolittle/sdk.execution';
+import { IArtifacts } from '@dolittle/sdk.artifacts';
+import { ExecutionContext } from '@dolittle/sdk.execution';
 import { Logger } from 'winston';
-import { Cancellation } from '@dolittle/sdk.resilience';
-import { FiltersClient } from '@dolittle/runtime.contracts/Runtime/Events.Processing/Filters_grpc_pb';
+import { Guid } from '@dolittle/rudiments';
+import { EventStore } from './EventStore';
+import { EventStoreClient } from '@dolittle/runtime.contracts/Runtime/Events/EventStore_grpc_pb';
+import { IEventStore } from './IEventStore';
 
 export type EventStoreBuilderCallback = (builder: EventStoreBuilder) => void;
 
 /**
  * Represents a builder for building event types, event handlers and event filters.
  */
 export class EventStoreBuilder {
```

Comment:
```suggestion
/**
 * Represents a builder for building an event store.
 */
export class EventStoreBuilder {
```
Just some old comments.